app/models/miq_server/worker_management/monitor.rb

```diff
@@ -147,8 +147,22 @@ def sync_needed?
       start_apache               if roles_changed &&  apache_needed?
 
       reset_queue_messages       if config_changed || roles_changed
+
+      update_sync_timestamp(@last_sync)
```

Comment:
I figured it wouldn't hurt to always set the timestamp on memcached even if the worker is still getting its messages from Drb.